west.yml

```diff
@@ -26,6 +26,102 @@ manifest:
 
   group-filter: [-babblesim, -optional]
 
+  venv:
+    name: ".venv"
+    definitions:
+      cipd_url: "https://chrome-infra-packages.appspot.com/dl"
+    bin-requirements:
```

Comment:
I take it this is meant as a sample not something that should ever be added, but I really do not like this idea. This should be brought up at architecture working group